ignite/pkg/xos/cp.go

```diff
@@ -0,0 +1,63 @@
+package xos
```

Comment:
Maybe this file could be called `copy.go` to be explicit?

Reply:
It follows the standard of the other files, move is mv, and remove is rm, like the commands. Do you think we should change all?